src/StructuredDiscussions/SDTopic.php

```diff
@@ -112,6 +112,15 @@ public function getModerationState(): string {
 		return $this->getRootRevision()['moderateState'] ?? AbstractRevision::MODERATED_NONE;
 	}
 
+	/**
+	 * Returns the topic's current lock state.
+	 *
+	 * @return string
+	 */
+	public function getLockStatus(): string {
+		return var_export($this->getRootRevision()['isLocked'], true);		
```

Comment:
This can just be `return $this->getRootRevision()['isLocked'];`.

Reply:
Done